src/main/java/com/google/sps/servlets/YoutubeServlet.java

```diff
@@ -39,25 +40,28 @@
   // Parameters required by YouTube API to retrieve the comment threads
   private static final String SNIPPET_PARAMETERS = "snippet,replies";
   private static final String ORDER_PARAMETER = "relevance";
-  // TODO: obtain actual API Key
-  private static final String DEVELOPER_KEY = "OUR_API_KEY";
+
   private static final String APPLICATION_NAME = "SAY";
+  private static final String DEVELOPER_KEY = "AIzaSyDYfjWcy1hEe0V7AyaYzgIQm_rT-9XbiGs";
```

Comment:
Maybe we can add a TODO to have developer key come from some other centralized config location?

Ideally, to make a change to configuration like these we wouldn't want to have to make a code change. (So that this kind of config isn't part of the java binary that gets packaged).

Reply:
How would you recommend obtaining the key? Do you mean storing it in a file such as pom.xml?

Reply:
Rick, I don't know if there is any configuration injection mechanism they can use(or easily make a separate config package vs. binary package). This seems trivial enough that having a separation could be an overkill.
It is fine to add a TODO but the priority is probably very low.